assets/src/edit-story/app/prepublish/guidance/media.js

```diff
@@ -134,3 +135,26 @@ export function videoElementLength(element) {
   }
   return undefined;
 }
+
+/**
+ * Check a if a video element's been optimized.
+ * If is not optimized, return guidance. Otherwise return undefined.
+ *
+ * @param {Element} element The element being checked
+ * @return {Guidance|undefined} The guidance object for consumption
+ */
+export function videoElementOptimized(element) {
+  const videoArea =
+    (element.resource.sizes.full?.height || 0) *
+    (element.resource.sizes.full?.width || 0);
+  const isLargeVideo = videoArea >= 1080 * 1920;
```

Comment:
Can we share/import this from the resolution settings used in `useFFmpeg.js`? E.g. I have a change to reduce the output res to 720p in #7335.

Reply:
Just added a shared constant for this. let me know if you'd like it named or used differently or anything